env.js

```diff
@@ -16,6 +16,8 @@ module.exports = function() {
     docLimit: safeNum(process.env.COUCH2PG_DOC_LIMIT),
     changesLimit: safeNum(process.env.COUCH2PG_CHANGES_LIMIT),
     retryCount: safeNum(process.env.COUCH2PG_RETRY_COUNT),
-    sleepMins: (safeNum(process.env.COUCH2PG_SLEEP_MINS) || 60)
+    sleepMins: (safeNum(process.env.COUCH2PG_SLEEP_MINS) || 60),
+    couchdbUsersMeta: process.env.COUCH2PG_USERS_META || false,
+    couchdbUsersMetaDocLimit: process.env.COUCH2PG_USERS_META_DOC_LIMIT || 50,
```

Comment:
Just use the `docLimit` from above. No need to have a separate variable.

Reply:
The docs in the users-meta database seem to be quite a bit larger than other docs possibly due to the presence of stack traces. This was causing the app to crash when I tested locally using docLimit so decided to have an additional parameter that can be set lower than the docLimit.

Reply:
Ok! Good call. We probably need to solve this for both cases so they automatically recover when this is set too high but that's outside of this issue.

Reply:
don't we need a safeNum safety check here?

Reply:
@nomulex Good spotting! As this has already been merged can you please raise a new issue for it?

Reply:
Ticket logged [here](https://github.com/medic/cht-couch2pg/issues/99) @garethbowen